### PR TITLE
docs: add VPC-related note to auth docs

### PR DESCRIPTION
**Pull request by padamstx**
Signed-off-by: Phil Adams <phil_adams@us.ibm.com>

### CODE REVIEW

**Comment by padamstx** [comment]
cc: @fode

File: ibm_cloud_sdk_core/token_managers/container_token_manager.py [hunk 1]
```diff
@@ -128,8 +128,8 @@ def retrieve_cr_token(self) -> str:
             with open(cr_token_filename, 'r', encoding='utf-8') as file:
                 cr_token = file.read()
             return cr_token
-        # pylint: disable=broad-except
         except Exception as ex:
+            # pylint: disable=broad-exception-raised
```
Comment (padamstx) [inline ibm_cloud_sdk_core/token_managers/container_token_manager.py:132]: For some unknown reason, we started seeing this new pylint error, and also for some unknown reason, we no longer get the "broad-except" warning :joy:

Comment (dpopp07) [reply ibm_cloud_sdk_core/token_managers/container_token_manager.py:132]: Weird - maybe a pylint version change?

**Review by dpopp07 — APPROVED** [review]

**Pull request merged**